cdap-data-fabric/src/test/java/co/cask/cdap/data/hbase/HBaseTestBase.java

```diff
@@ -55,7 +55,16 @@ public String getZkConnectionString() {
 
   // Test startup / teardown
 
-  public abstract void startHBase() throws Exception;
+  public final void startHBase() throws Exception {
+    // Tune down the connection thread pool size
+    getConfiguration().setInt("hbase.hconnection.threads.core", 5);
+    getConfiguration().setInt("hbase.hconnection.threads.max", 10);

```

Comment:
Can add that too. From the testing I conducted, there are far more HConnection threads than HTable threads (probably because we are closing HTable connections during test, but not HConnection. Also, the HTable thread pool has one core thread, while the HConnection thread pool has 256 core threads).